File: C/1074-par-ou-impar.cpp
#include <stdio.h>
 
int main() {
 
	int A, B, C;
	
	 scanf ("%i", &A);
			 	 
	 for (B = 1; B <= A; ++B){
	 	
	 scanf ("%i", &C);
	
	 if ( C == 0){
	 	printf ("NULL\n");
	 }
	 	 
	if (C%2 != 0){
	 	if(C > 0){
	 	 printf ("ODD POSITIVE\n");	
		}
		else {
		
	 	printf ("ODD NEGATIVE\n");
	    }
	    
	 }
		
	 if (C % 2 == 0 && C != 0){
	 	
	 	if (C > 0) {
	 	  printf ("EVEN POSITIVE\n");
		}
	 	if ( C < 0){
	 		printf ("EVEN NEGATIVE\n");
		 }	 	
	 }	 
						
	}
 
    return 0;
}

File: Python/1074-par-ou-impar.py
numero = int(input())

for i in range(0,numero, 1):
    contador = int(input())
    if contador%2==0 and contador<0:
        print('EVEN NEGATIVE')
    elif contador%2==0 and contador>0:
        print('EVEN POSITIVE')
    elif contador%2!=0 and contador<0:
        print('ODD NEGATIVE')
    elif contador%2!= 0 and contador>0:
        print('ODD POSITIVE')
    elif contador == 0:
        print('NULL')